projects/plugins/boost/.eslintrc.cjs

```diff
@@ -1,11 +1,19 @@
-// eslint-disable-next-line import/no-extraneous-dependencies
+// eslint-disable-next-line import/no-extraneous-dependencies, @typescript-eslint/no-var-requires
 const loadIgnorePatterns = require( 'jetpack-js-tools/load-eslint-ignore.js' );
 
+/**
+ * @type {import("eslint").Linter.Config}
+ */
 module.exports = {
 	root: true,
 	extends: [
 		require.resolve( 'jetpack-js-tools/eslintrc/base' ),
 		require.resolve( 'jetpack-js-tools/eslintrc/wp-eslint-plugin/recommended' ),
+		/**
+		 * we need to add our TS specific config
+		 * in order to extend 'plugin:@typescript-eslint/recommended'
+		 * Otherwise svelte TS files won't be parsed
+		 */
```

Comment:
Then instead of that we can (and we need to) extend typescript config in svelte config in order to apply TS rules for ESLint and I think that's a better approach.